code/modules/projectiles/guns/launcher/pneumatic.dm

```diff
@@ -76,11 +76,12 @@
 /obj/item/gun/launcher/pneumatic/attack_self(mob/user)
 	eject_tank(user)
 
-/obj/item/gun/launcher/pneumatic/consume_next_projectile(mob/user=null)
+/obj/item/gun/launcher/pneumatic/consume_next_projectile(atom/movable/firer)
 	if(!item_storage.contents.len)
 		return null
 	if (!tank)
-		to_chat(user, "There is no gas tank in [src]!")
+		if(isliving(firer))
```

Comment:
As far as I know it's harmless to pass strings to non-mobs, it just doesn't do anything.